package.json

```
@@ -2,65 +2,38 @@
     "name": "user-profile-app",
     "version": "31.0.0",
     "description": "User Profile App for DHIS2",
-    "main": "src/index.html",
+    "license": "BSD-3-Clause",
     "scripts": {
-        "coverage": "babel-node node_modules/.bin/isparta cover --root ./src --report text --report html --report lcov node_modules/mocha/bin/_mocha -- --reporter dot --require config/setup.js --recursive test",
-        "prebuild": "rm -rf build && mkdir build && d2-manifest package.json manifest.webapp",
-        "build": "webpack",
-        "postbuild": "cp -r index.html icon.png auth.png node_modules/babel-polyfill/dist/polyfill.min.js manifest.webapp i18n build/",
-        "test": "echo Everything is probably ok! # mocha --reporter spec --compilers js:babel-core/register --require config/setup.js --recursive test",
-        "test-watch": "echo Everything is always ok! # mocha --reporter spec --compilers js:babel-core/register --require config/setup.js --recursive test --watch",
-        "lint": "eslint --fix src/",
-        "validate": "npm ls",
-        "prestart": "d2-manifest package.json manifest.webapp",
-        "start": "webpack-dev-server"
+        "build": "d2-app-scripts build",
+        "start": "d2-app-scripts start",
+        "test": "d2-app-scripts test",
+        "deploy": "d2-app-scripts deploy",
+        "lint": "d2-style js check && d2-style text check",
```

Some extra scripts used by other apps:

```suggestion
        "lint:js": "d2-style js check",
        "lint:text": "d2-style text check",
        "lint:staged": "yarn lint:js --staged && yarn lint:text --staged",
        "lint": "yarn lint:js && yarn lint:text",
        "format:js": "d2-style js apply",
        "format:text": "d2-style text apply",
        "format:staged": "yarn format:js --staged && yarn format:text --staged",
```

---

Looks good 👍